__test__/slackify-markdown.test.js

```diff
@@ -7,7 +7,7 @@ test('Simple text', () => {
 });
 
 test('Escaped text', () => {
-  expect(slackifyMarkdown('*h&ello>world<')).toBe('*h&amp;ello&gt;world&lt;\n');
+  expect(slackifyMarkdown('*h&ello>world<')).toBe('\\*h&amp;ello&gt;world&lt;\n');
```

Comment:
`remark-stringify@9` seems to be aggressive in escaping some entities, [like this `*`](https://github.com/syntax-tree/mdast-util-to-markdown/blob/a57fe5cc57fb6ef1162962033e55cbb31991418f/lib/unsafe.js#L53). I'll look into whether this causes any issues in Slack.

Reply:
Looks like Slack `invalid_payload`s an escaped asterisk `\*`, so I've skipped the [`safe` escaping](https://github.com/syntax-tree/mdast-util-to-markdown/blob/main/lib/util/safe.js) for now.